### PR TITLE
Added this week in os #15

### PR DESCRIPTION
**Pull request by emmasofiah2o**
- Created file
- Added og
- Updated config

### CODE REVIEW

File: _posts/2022-09-05-this-week-in-os-15.md [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+  'aws-lambda' as devDependency ([@oscard0m])
+
+## Unknown
+
+- [LukeMathWalker/tracing-bunyan-formatter]
+  [#20](https://github.com/LukeMathWalker/tracing-bunyan-formatter/pull/20)
+  Migrate from `claim` to `claims` ([@Turbo87])
+- [Turbo87/segelflug-classifieds]
+  [#350](https://github.com/Turbo87/segelflug-classifieds/pull/350) Fix "byte
+  index is not a char boundary" crash ([@Turbo87])
```
Comment (Turbo87) [inline _posts/2022-09-05-this-week-in-os-15.md]: both are "Rust"

File: _posts/2022-09-05-this-week-in-os-15.md [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+  section for 'Other examples' ([@oscard0m])
+- [probot/example-vercel]
+  [#49](https://github.com/probot/example-vercel/pull/49) docs(readme): add
+  considerations for deploying with vercel ([@oscard0m])
+- [probot/probot] [#1724](https://github.com/probot/probot/pull/1724)
+  build(deps): upgrade octokit/types to v7.1.1 ([@oscard0m])
+- [soxhub/formatjs] [#2](https://github.com/soxhub/formatjs/pull/2) fix: the
+  same description inside nested html should have equal ids ([@candunaj])
+- [volta-cli/action] [#100](https://github.com/volta-cli/action/pull/100) Fix
+  changelog heading ([@Turbo87])
```
Comment (Turbo87) [inline _posts/2022-09-05-this-week-in-os-15.md]: JavaScript

File: _posts/2022-09-05-this-week-in-os-15.md (hunk 1 above)
Comment (emmasofiah2o) [reply _posts/2022-09-05-this-week-in-os-15.md]: Awesome! Thanks

File: _posts/2022-09-05-this-week-in-os-15.md [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+  ([@oscard0m])
+- [oscard0m/gravity_dummy]
+  [#3](https://github.com/oscard0m/gravity_dummy/pull/3) feat(index): cowsay
+  library ([@oscard0m])
+- [oscard0m/gravity_dummy]
+  [#2](https://github.com/oscard0m/gravity_dummy/pull/2) build(package-lock):
+  add missing package-lock ([@oscard0m])
+- [oscard0m/gravity_dummy]
+  [#1](https://github.com/oscard0m/gravity_dummy/pull/1) Update index.ts
+  ([@oscard0m])
```
Comment (oscard0m) [inline _posts/2022-09-05-this-week-in-os-15.md]: We can remove these ones:

```suggestion
```

File: _posts/2022-09-05-this-week-in-os-15.md [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+- [oscard0m/octoherd-script-fix-test-workflow-octokit]
+  [#1](https://github.com/oscard0m/octoherd-script-fix-test-workflow-octokit/pull/1)
+  feat: Initial version ([@oscard0m])
+- [oscard0m/octoherd-script-remove-renovate-package.json]
+  [#1](https://github.com/oscard0m/octoherd-script-remove-renovate-package.json/pull/1)
+  feat: Initial version ([@oscard0m])
```
Comment (oscard0m) [inline _posts/2022-09-05-this-week-in-os-15.md]: section: Octokit

File: _posts/2022-09-05-this-week-in-os-15.md [hunk 5]
```diff
@@ -0,0 +1,285 @@
...
+- [probot/example-vercel]
+  [#50](https://github.com/probot/example-vercel/pull/50) docs(README): add
+  section for 'Other examples' ([@oscard0m])
+- [probot/example-vercel]
+  [#49](https://github.com/probot/example-vercel/pull/49) docs(readme): add
+  considerations for deploying with vercel ([@oscard0m])
+- [probot/probot] [#1724](https://github.com/probot/probot/pull/1724)
+  build(deps): upgrade octokit/types to v7.1.1 ([@oscard0m])
```
Comment (oscard0m) [inline _posts/2022-09-05-this-week-in-os-15.md]: section: Probot

if we don't want to create a new section, Octokit can be good as well.

Comment (emmasofiah2o) [reply _posts/2022-09-05-this-week-in-os-15.md]: I am making the changes now. What about the vercel PR?

File: _posts/2022-09-05-this-week-in-os-15.md (hunk 3 above)
Comment (emmasofiah2o) [reply _posts/2022-09-05-this-week-in-os-15.md]: Oh it's here, nvm ahah

File: config/this-week-in-open-source.config.json [hunk 6]
```diff
@@ -43,7 +43,9 @@
         "oscard0m/octoherd-script-migrate-pika",
         "oscard0m/octoherd-script-replace-all-repository-topics",
         "oscard0m/octoherd-script-watch-unwatch-or-ignore",
-        "marcoow/chat-server"
+        "marcoow/chat-server",
+        "oscard0m/gravity_dummy",
+        "oscard0m/example-vercel-ts]"
```
Comment (oscard0m) [inline config/this-week-in-open-source.config.json]:
```suggestion
        "oscard0m/example-vercel-ts"
```

File: _posts/2022-09-05-this-week-in-os-15.md [hunk 7]
```diff
@@ -0,0 +1,280 @@
...
+
+## Unknown
+
+- [soxhub/formatjs] [#2](https://github.com/soxhub/formatjs/pull/2) fix: the
+  same description inside nested html should have equal ids ([@candunaj])
```
Comment (BobrImperator) [inline _posts/2022-09-05-this-week-in-os-15.md]: I think we can ignore it.
`soxhub` is a client organization and this package is not quite, not exactly open source. It's weird ;D

File: config/this-week-in-open-source.config.json [hunk 8]
```diff
@@ -43,7 +43,9 @@
         "oscard0m/octoherd-script-migrate-pika",
         "oscard0m/octoherd-script-replace-all-repository-topics",
         "oscard0m/octoherd-script-watch-unwatch-or-ignore",
-        "marcoow/chat-server"
+        "marcoow/chat-server",
```
Comment (BobrImperator) [inline config/this-week-in-open-source.config.json:47]:
```suggestion
        "marcoow/chat-server",
        "soxhub/formatjs",
```

File: _posts/2022-09-05-this-week-in-os-15.md (hunk 7 above)
Comment (BobrImperator) [inline _posts/2022-09-05-this-week-in-os-15.md]:
```suggestion
```

**Review by Mikek2252 — APPROVED** [review]
Looks fine to me and everything else seems resolved

**Review by oscard0m — APPROVED** [review]

**Pull request merged**